eng/common/Update-Change-Log.ps1

```diff
@@ -46,8 +47,8 @@ function Get-VersionTitle($Version, $Unreleased)
    # Generate version title
    $newVersionTitle = "## $Version $UNRELEASED_TAG"
    if ($Unreleased -eq $False) {
-      $releaseDate = Get-Date -Format "(yyyy-MM-dd)"
-      $newVersionTitle = "## $Version $releaseDate"
+      $actualReleaseDate = $ReleaseDate ?? (Get-Date -Format "yyyy-MM-dd");
```

Comment:
`??` is very new. PS7 and PS6 won't support it. Instead, just do something like:

```powershell
if (!$ReleaseDate) {
  $ReleaseDate = Get-Date
}
```

And keep it a date until you need to format it so it's validated, like as a parameter.

Reply:
You can also default the parameter initially like so:

```powershell
[VAlidateNotNullOrEmpty()]
[DateTime] $ReleaseDate = Get-Date
```

Reply:
I use the existence of this parameter as part of the logic so default value won't work but you are right about `??` being too new.